.lintstagedrc

```diff
@@ -0,0 +1,4 @@
+{
+    "*": "prettier --write",
+    "*.(ts|tsx)": ["eslint --fix"]
```

Comment:
Hmm, I wonder if we want the hook to change the file. I'm expecting the hook to raise an error if there is a linting error, not to try to fix it.

Reply:
This is consistent with the hooks in element-web & matrix-js-sdk